aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/SigV4AuthDecorator.kt

```diff
@@ -65,6 +70,13 @@ class SigV4AuthDecorator : ClientCodegenDecorator {
         baseCustomizations: List<ConfigCustomization>,
     ): List<ConfigCustomization> =
         baseCustomizations + SigV4SigningConfig(codegenContext.runtimeConfig, codegenContext.serviceShape.getTrait())
+
+    override fun extras(codegenContext: ClientCodegenContext, rustCrate: RustCrate) {
+        if (codegenContext.serviceShape.supportedAuthSchemes().contains("sigv4a")) {
+            // Add optional feature for SigV4a support
+            rustCrate.mergeFeature(Feature("sigv4a", false, listOf("aws-runtime/sigv4a")))
+        }
+    }
```

Comment:
should we enable sigv4a by default?

Reply:
I'll make it default unless it conflicts with WASM.